### PR TITLE
feat(Usage Reports): add billing snapshot config service to usage reports

### PR DESCRIPTION
**Pull request by manukm-ibm**
## PR summary
Introducing new CRUD APIs to configure billing reports snapshot.
TRI: https://github.ibm.com/BSS/architecture/blob/master/design-docs/services/billing/apptio-integration.md

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by padamstx** [comment]
The commit message `feat(global catalog): add global-catalog service to project` doesn't make sense.  Is this a copy/paste error?  If yes, then no need to fix this as I'll do a squash and merge when the PR is ready.

**Comment by manukm-ibm** [comment]
> The commit message `feat(global catalog): add global-catalog service to project` doesn't make sense. Is this a copy/paste error? If yes, then no need to fix this as I'll do a squash and merge when the PR is ready.

Yes Phil. This was a copy paste issue.

File: modules/usage-reports/src/test/java/com/ibm/cloud/platform_services/usage_reports/v4/UsageReportsIT.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright IBM Corp. 2020.
+ * (C) Copyright IBM Corp. 2023.
```
Comment (padamstx) [inline modules/usage-reports/src/test/java/com/ibm/cloud/platform_services/usage_reports/v4/UsageReportsIT.java:2]: Same comment as in the Python PR.

**Review by padamstx — CHANGES_REQUESTED** [review]
Same issue with the integration test changes here as in the python PR.  You'll need to revert your changes to the UsageReportsIT.java file, then regenerate the int tests off to the side, then copy/paste the test code blocks for the new operations into the checked-in copy of the integration tests.  Same with the working examples code.

**Pull request closed without merging**